app/views/journals/show.html.erb

```diff
@@ -1,52 +1,133 @@
+        <div><%= t('entry.table.updated_at') %></div>
+        <div><%= t('entry.table.created_at') %></div>
+      </li>
+      <hr>
+      <% @journal.journal_entries.each do |entry| %>
+        <li class="table-row clickable-row" data-href="<%= user_journal_journal_entry_path(@user, @journal, entry) %>">
+          <div class="id"><%= entry.id %></div>
+          <div class="row-title"><%= entry.title %></div>
+          <div><%= entry.updated_at.to_formatted_s(:short) %></div>
+          <div><%= entry.created_at.to_formatted_s(:short) %></div>
+          <div>
+            <div class="delete-button">
+              <%= link_to user_journal_journal_entry_path(@user, @journal, entry), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: t('user.message.confirm')
+                  } do %>
+                <img src="<%= asset_path('delete.svg') %>" alt="Add Entry" style="width: 25px; height: 25px;"></img>
+              <% end %>
+            </div>
+          </div>
+        </li>
+      <% end %>
+    </ul>
+  </div>
 </div>
-
-<div class="container">
-  <div class="row">
-    <div class="col-12">
-      <table class="table table-striped">
-        <thead class="thead-dark">
-          <tr>
-            <th scope="col">ID</th>
-            <th><%= link_to t('entry.table.title'), sort: :title %> </th>
-            <th><%= link_to t('entry.table.updated_at'), sort: :updated_at %> </th>
-            <th><%= link_to t('entry.table.created_at'), sort: :created_at %> </th>
-            <th scope="col"></th>
-          </tr>
-        </thead>
-        <tbody>
-          <% @journal.journal_entries.each do |entry| %>
-          <tr>
-            <td><%= entry.id %></td>
-            <td><%= entry.title %></td>
-            <td><%= entry.updated_at.to_formatted_s(:short) %></td>
-            <td><%= entry.created_at.to_formatted_s(:short) %></td>
-            <td>
-            <%= link_to t('entry.action.show'), user_journal_journal_entry_path(@user,@journal,entry), class: "btn btn-primary"%>
-
-            <%= link_to t('entry.action.delete'), user_journal_journal_entry_path(@user,@journal,entry), class: "btn btn-danger", data: {
+<div class="information">
+  <div class="description">
+    <div class="item-title">
+      <h3><%= t('journal.table.description') %></h3>
+    </div>
+    <div class="description-content">
+      <hr style="margin:2%;">
+      <p>
+        <%= @journal.description %>
+      </p>
+    </div>
+  </div>
+  <div class="information-files-attached">
+    <div class="item-title">
+      <h3><%= t('file.attached') %></h3>
+    </div>
+    <hr style="margin:2%;">
+    <div class="files-items">
+      <% @journal.files.each do |file| %>
+        <div class="file">
+          <div>
+            <img src="<%= asset_path('image.svg') %>" alt="Edit Entry" style="width: 25px; height: 25px;"></img>
+          </div class="file-filename">
```

Comment:
isnt there a mistake with the file name div ? im not shure but i belive it should look like this: 

```
 <div>
    <img src="<%= asset_path('image.svg') %>" alt="Edit Entry" style="width: 25px; height: 25px;">
 </div>
 <div class="file-filename">
    <%= link_to file.filename, file %>
 </div>
```